src/lib/rest/httpRequestSend.cpp

```diff
@@ -168,6 +168,7 @@ int httpRequestSendWithCurl
    const std::string&     orig_content_type,
    const std::string&     content,
    const std::string&     fiwareCorrelation,
+   const std::string&     notifyFormat,
```

Comment:
I was aboput to do that but decided not to change this function to have it more generic.
The way it works right now is that whatever text is found in 'notifyFormat' is sent as X-Ngsiv2-AttrsFormat (exceptions: if empty or JSON, the header is omitted).

To change this would be to complicate the function and at the same time make it less generic.

I'm not saying NO, but right now it doesn't seem like a good idea.